build/dependencies.props

```diff
@@ -14,6 +14,8 @@
     <NpgsqlVersion>3.2.0-*</NpgsqlVersion>
     <NpgsqlEntityFrameworkCorePostgreSQLVersion>1.*</NpgsqlEntityFrameworkCorePostgreSQLVersion>
 
+    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
Didn't realize it at first. Can you group it together with the properties at the top of the file? That's where it is in other `dependencies.props` files.